### DISCUSSION TITLE
Center header

### DISCUSSION THREAD

**Discussion by pothitos** [marp-team/marp#394] [Q&A] [answered]
Hello!

How can we center a header?

Thanks a lot for Marp!

**Comment by yhatt** [comment, marked as answer]
```html
<style>
header {
    right: 30px;
    text-align: center;
}
</style>
```

When using `default` theme, the header has the anchored positioning only to the left. By adding it to the right, the header element will become having full width and `text-align: center` will be working within that.

Related: #47
